docs/migration/2_0.md

```diff
@@ -185,6 +185,7 @@ Support for functions dropped:
 - `CartItemListComponent` no longer requires `FormBuilder`, `FeatureConfigService` and  `CartService`. `CartService` was replaced with corresponding methods from `ActiveCartService`. The deprecated method `isSaveForLaterEnabled()` was removed.
 - `CartItemComponent` no longer uses `FeatureConfigService`. The deprecated method `isSaveForLaterEnabled()` was removed.
 - `PaymentFormComponent` requires new parameter now: `UserAddressService`.
+- `CartCouponComponent` no longer requires `AuthService` and `FeatureConfigService`.
```

Comment:
Could you merge it with existing info (line 161) about this component?

Reply:
done